package.json

```diff
@@ -14,9 +14,11 @@
     "@babel/cli": "^7.23.4",
     "@babel/core": "^7.23.7",
     "@babel/preset-env": "^7.23.8",
+    "@rollup/plugin-typescript": "^11.1.6",
     "acorn": "^8.11.3",
     "google-closure-compiler": "20230802.0.0",
-    "html-minifier-terser": "7.2.0"
+    "html-minifier-terser": "7.2.0",
+    "rollup": "^4.12.0"
```

Comment:
There's no script checked into the repo that actually uses roll-up, so it probably doesn't need to be added to the dependencies.

Reply:
Oh I tree shake this using rollup

Reply:
Yes, but because there isn't any actual script in the repository that uses it, it doesn't need to be in package.json. Instead it's adding these packages for everyone who installs it, for no actual usefulness.